src/code42cli/cmds/alerts.py

```diff
@@ -274,7 +274,10 @@ def send_to(
     or_query,
     **kwargs
 ):
-    """Send alerts to the given server address."""
+    """Send alerts to the given server address.
+
+    HOSTNAME format: address:port where port is optional and defaults to 514.
```

Comment:
I could have used a constant here but that was not being rendered in docs so had to copy them in all places.

Reply:
Instead of putting this in code here, can he make the help text in `server_options` instead? That way it's documented both in code and in the terminal, and doesn't need to be copied to multiple places.

Reply:
nevermind, I now see that `click.argument` doesn't have a `help` param, and that this is the advised way to do this.